# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../blender/compile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

So far every version of blender needed one or the other compile tweak, just
drop it into this file ,-)

  - Rene Rebe <rene@exactcode.de>

--- blender2.42/config/linux2-config.py.vanilla	2006-07-17 10:31:05.000000000 +0200
+++ blender2.42/config/linux2-config.py	2006-07-17 10:31:37.000000000 +0200
@@ -22,7 +22,7 @@
 WITH_BF_OPENEXR = 'true'
 BF_OPENEXR = '/usr'
 BF_OPENEXR_INC = '${BF_OPENEXR}/include/OpenEXR'
-BF_OPENEXR_LIB = 'Half IlmImf Iex Imath '
+BF_OPENEXR_LIB = 'Half IlmImf Iex Imath z'
 
 WITH_BF_JPEG = 'true'
 BF_JPEG = '/usr'
